tests/fixtures/issues/issue-900.php

```diff
@@ -6997,59 +6997,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-900-ED' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 5.1; en; rv:1.9a1) Gecko/20061128 BonEcho/0.7b1',
```

Comment:
I removed this from the test suite because the versioning here doesn't make sense. "BonEcho" was the code name for Firefox 2 when in development, and all other BonEcho strings I've seen have had a 2.x version number here.

Also, I think if this were parsed as Firefox 0.7, it would be assumed that this is an early version of Firefox, but as seen from the Gecko build number (`20061128`) this is more on-par with when Firefox 2.0 was released: https://en.wikipedia.org/wiki/History_of_Firefox#Version_2